text/0028-csv-indexation.md

```diff
@@ -0,0 +1,156 @@
+- Title: Csv Indexation
+- Start Date: 2021-04-9
+- Specification PR: [PR-#28](https://github.com/meilisearch/specifications/pull/28)
+- MeiliSearch Issues: [#128](https://github.com/meilisearch/transplant/issues/128),[#1332](https://github.com/meilisearch/MeiliSearch/issues/1332)
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+The initiation step of document indexing is to send some file matching a format to be parsed and tokenized in order to give search results to end-users. A [CSV](https://en.wikipedia.org/wiki/Comma-separated_values) file format is broadly used to store and exchange data in a simple format.
```

Comment:
Maybe kero will explain this better than me, but I'm not sure there is any tokenization during indexation.
From my POV the indexation purpose is to sort/organize the documents of the user to make MeiliSearch able to retrieve them in the most efficient way during a search.

Reply:
Yes, I wrote that to point out why we need to send data for indexing. I can just write that we need indexing to return those data as a search result.